src/cascadia/TerminalSettingsEditor/ViewModelHelpers.h

```diff
@@ -48,7 +48,7 @@ public:                                                   \
         if (name() != value)                              \
         {                                                 \
             target.name(value);                           \
-            _NotifyChanges(L"Has" #name, L#name);         \
+            _NotifyChanges(L"Has" L#name, L#name);        \
```

Comment:
due to how the compiler stitches strings together, this is not a problem.

Reply:
(why did this need to change?)

Reply:
For whatever reason, Base Layer was not updating the UI when the SettingContainer reset button was clicked. I tracked the bug down to the macros (which was surprising, because the non-base layer profile pages worked just fine). Worked with @miniksa to figure out what was wrong with the macros and these changes fixed it. 🤷‍♂️

We think the missing 'L' might've improperly concatenated "Has" and \<name\>, and the notification that gets fired is silently ignored because nothing matches the garbled up "Has\<Name\>".

Reply:
Hmm. That's definitely not it -- `L"X" "Y"` is identical to `L"XY"` and `L"X" L"Y"`

Reply:
well then it was probably the other thing where you had a `#Name`, Carlos, that wasn't getting substituted properly and it got fixed changing it to `#name`?

Reply:
Yeah I bet _that_ got substituted as `"Name"` (not expanding the macro!)

Reply:
Why didn't that result in a compiler error though? That's where I'm confused.

Reply:
In the C preprocessor's language, `#X` means "turn X into a string" (=> `"X"`). If X is a macro parameter, it is expanded. If it's not, the literal word just gets turned into a string.

Reply:
OH. OH. IT WORKS BECAUSE profile _HAS_ a `HasName`. I was thinking about `#`, not `##`.

Reply:
Omg, that makes so much sense! Alright, I'll revert the 'L' changes then.

Reply:
To be clear, I had no idea why it wasn't working. I was literally just playing spot the difference between the macro you said was broken and the code you said was working.... :P